### PR TITLE
feat: Pasting & dropping files

### PR DESCRIPTION
**Pull request by dcadavez66**
Added support to paste images from the clipboard into the editor

### CODE REVIEW

File: packages/core/src/api/parsers/handleFileInsertion.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+          fileBlockType = fileBlockConfig.type;
+          break;
+        }
+      }
+    }
+
+    const file = items[i].getAsFile();
+    if (file) {
+      editor.uploadFile(file).then((updateData) => {
+        // TODO: Why can `uploadFile` return an object again?
```
Comment (YousefED) [inline packages/core/src/api/parsers/handleFileInsertion.ts:85]: See comments of `uploadFile`

File: packages/core/src/api/parsers/handleFileInsertion.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        if (checkMIMETypesMatch(items[i].type, mimeType)) {
+          fileBlockType = fileBlockConfig.type;
+          break;
+        }
+      }
+    }
+
+    const file = items[i].getAsFile();
+    if (file) {
+      editor.uploadFile(file).then((updateData) => {
```
Comment (YousefED) [inline packages/core/src/api/parsers/handleFileInsertion.ts:84]: use async / await?

**Review by YousefED — COMMENTED** [review]
nice! small comments

**Comment by lamtuanamc** [comment]
@dcadavez66
 I can't paste images after using Cmd X, Cmd C is normal

**Pull request merged**